content/en/threat-model.md

```diff
@@ -0,0 +1,123 @@
+---
+title: 'Threat Model'
+description: ''
+category: 'About sigstore'
+position: 3
+---
+
+We provide a threat model that details the impact of a compromise in any piece of the Sigstore ecosystem. Based on this assessment, we discuss existing mitigations and provide recommendations about the secure configuration of Sigstore. We find that the TUF root of trust provides strong revocation abilities for all Sigstore services in the event of a compromise, but OIDC accounts and issuers used with Sigstore do not have the same revocation abilities. Further, identity and consistency monitoring are critical to many of the security claims of Sigstore, but they are not yet widely applied in practice. We recommend an increased focus on monitoring and 2fa for all OIDC accounts used with Sigstore.
```

Comment:
Nitpick: 2FA is conventionally capitalized 🙂 

(There are other instances below, as well.)

```suggestion
We provide a threat model that details the impact of a compromise in any piece of the Sigstore ecosystem. Based on this assessment, we discuss existing mitigations and provide recommendations about the secure configuration of Sigstore. We find that the TUF root of trust provides strong revocation abilities for all Sigstore services in the event of a compromise, but OIDC accounts and issuers used with Sigstore do not have the same revocation abilities. Further, identity and consistency monitoring are critical to many of the security claims of Sigstore, but they are not yet widely applied in practice. We recommend an increased focus on monitoring and 2FA for all OIDC accounts used with Sigstore.
```